src/m365/spo/commands/web/WebProperties.ts

```diff
@@ -39,4 +42,18 @@ export interface CurrentChangeToken {
 
 export interface ResourcePath {
   DecodedUrl: string;
+}
+export interface AssociatedGroupProperties {
```

Comment:
Since we're not doing anything with this type in our code, we don't need to define it and can simplify our code maintenance

Reply:
Even I was skeptical in adding the interfaces. But then I thought, maybe we can use that if we extend in future if we want to set `AssociatedGroup `using `spo web set`. But I am with you on that. For now let us keep it simple and extend it later if and when needed. 👍